docs/dev/development.adoc

```diff
@@ -180,13 +180,20 @@ NOTE: Refer https://github.com/golang/go/wiki/LearnTesting for Go best practices
 
 *Prerequisites:*
 
-* A `minishift` or OpenShift environment with Service Catalog enabled:
+* A `minishift` or OpenShift 3.11 environment with Service Catalog enabled:
 +
 ----
 $ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--enable=*,service-catalog,automation-service-broker,template-service-broker"
 ----
+OR
+* A `crc` environment for 4.* local cluster:
++
+For `crc` installation please refer https://github.com/code-ready/crc#documentation
++
+OR
+* A 4.* cluster hosted remotely
 
-* `odo` and `oc` binaries in `$PATH`.
+NOTE: Make sure that `odo` and `oc` binaries are in `$PATH`. Use the clone odo directory to launch test on 3.11 and 4.* cluster. To run test 3.11 cluster locally no cluster configuration is needed, however a 4.* cluster needs to be configured before launching the test against it. The files `kubeadmin-password` and `kubeconfig` which contains cluster login details should be present in the auth directory and it should reside parallel to the `Makefile`. If it is not present in the auth directory, please create it. Then run `make configure-installer-tests-cluster` to configure the 4.* cluster.
```

Comment:
```
NOTE: Make sure that `odo` and `oc` binaries are in `$PATH`. Use the clone odo directory to launch the tests on 3.11 and 4.* clusters. To run the tests on a 3.11 cluster locally 
```

Reply:
> To run test 3.11 cluster locally no cluster configuration is needed,

I think that this is true only if you are using Minishift or `oc cluster up`.

Reply:
yes, you are right. So any phrase you want me to update instead. For example
```
No cluster configuration is needed to run test on 3.11 cluster spin up by `Minishift` or `oc cluster up` locally.
```

Reply:
Updated. @kadel PTAL